src/main/java/edu/ucsb/cs156/organic/Interceptors/RoleUserInterceptor.java

```diff
@@ -1,4 +1,4 @@
-package edu.ucsb.cs156.organic.interceptors;
+package edu.ucsb.cs156.organic.Interceptors;
```

Comment:
NOOOOO!!! 

We already fixed this!
Don't bring back this bad code that is unrelated to your changes!